subjects/index.js

```diff
@@ -0,0 +1,40 @@
+const pg = require('pg');
+
+function executeSQL(context, params) {
+    const connectionString = 'postgres://postgres:postgres@localhost:5432/sample_db'
+    const client = new pg.Client({ connectionString })
+
+    client.connect();
+    const { maxLat, minLat, maxLon, minLon } = params
+
+    if (!maxLat || !minLat || !maxLon || !minLon) {
+        context.res = {
+            status: 422,
+            body: 'Query params must include maxLat, minLat, maxLon, and minLon'
+        }
+        return context.done()
+    }
+
+    const query = `SELECT * FROM subjects where lat < ${maxLat} and lat > ${minLat}
+        and lon < ${maxLon} and lon > ${minLon}`;
+
+    client.query(query)

```

Comment:
Another piece that you could extract into its own function (e.g. `performQuery`) if you want to. I personally would extract it out; it also looks (nearly?) exactly the same as the `client.query` performed in the temperature/index.js file, would be nice to DRY it up.